src/Enhavo/Bundle/MediaBundle/composer.json

```diff
@@ -19,7 +19,9 @@
     "spatie/image-optimizer": "^1.0",
     "imagine/imagine": "^1.2",
     "guzzlehttp/guzzle": "^6.5",
-    "symfony/mime": "^5.4"
+    "symfony/mime": "^5.4",
+    "ext-gd": "*",
```

Comment:
Ist gd mandatory? Weil könnte ja auch mit imagick gehen und dann bräuchte man gd nicht

Reply:
das letzte fallback ist GD:

        } else {
            $imagine = new Imagine();
            $imagine = $imagine->load($content->getContent());
            $imagine = $this->resize($imagine, $setting);
            $imagine->save($content->getFilePath(), $this->getSaveOptions($format, $setting));
        }
        
        ggf müssten wir es dann auch noch mit einre classexists abfrage ausstatten?